base/system/services/rpcserver.c

```diff
@@ -6735,7 +6750,7 @@ RI_ScValidatePnPService(
     if (!RtlAreAllAccessesGranted(hManager->Handle.DesiredAccess,
                                   SC_MANAGER_CONNECT))
     {
-        DPRINT1("No SC_MANAGER_CONNECT access!\n");
+        DPRINT("No SC_MANAGER_CONNECT access!\n");
```

Comment:
Last nitpick (2 of 2) before I will signal my approval:
```suggestion
        DPRINT("No SC_MANAGER_CONNECT access\n");
```

Reply:
Changed. Thanks.